src/Sylius/Bundle/ApiBundle/Resources/config/routing/product_variant.yml

```diff
@@ -1,19 +1,19 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
-sylius_api_product_variant_index:
-    path: /
-    methods: [GET]
-    defaults:
-        _controller: sylius.controller.product_variant:indexAction
-        _sylius:
-            paginate: $limit
-            serialization_version: $version
-            grid: sylius_api_product_variant
-            section: api
-            serialization_groups: [Default]
-            criteria:
-                product: $productId
+sylius_api_product_variant:
```

Comment:
You should also remove prefix from main.yml file, and add this prefix to the rest of router paths.